src/wazuh_mcp_server/utils/validation.py

```
@@ -135,57 +137,99 @@ def __init__(self, **data):
         return ""
     
+    # Convert to string if not already
+    if not isinstance(input_str, str):
+        input_str = str(input_str)
+    
     # Remove null bytes and control characters
     sanitized = re.sub(r'[\x00-\x08\x0B\x0C\x0E-\x1F\x7F]', '', input_str)
     
+    # Remove potentially dangerous characters for command injection
+    sanitized = re.sub(r'[;&|`$<>]', '', sanitized)
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)